index.html

```diff
@@ -0,0 +1,303 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title> mobile version</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="header">
+        <a href="#" class="logo"> My Logo </a>
+        <nav>
+          <button type="button" class="buto">
+            <img alt="nav" class="menu" src="./images/Union.png" />
+          </button>
+        </nav>
+      </header>
+  
+      <section class="flex-container">
+        <div class="intro">
+          <h1>I'm Loic<br />Glad to see<br />you</h1>
+        </div>
+        <p class="introduction">
+          I'm software developer! I can help you<br />to build a product, feature
+          or website look<br />through some of my work and<br />
+          experience! If you like what you see and<br />have a project you need to
+          coded, don't<br />hesitate to contact me.
+        </p>
+        <h4 class="connection">LET'S CONNECT</h4>
+        <div class="social">
+          <ul>
+            <li>
+              <a href="#"> <img alt="twitter" src="./images/twitter.png" /> </a>
+            </li>
+            <li>
+              <a href="#"> <img alt="linkedin" src="./images/linkedin.png" /></a>
+            </li>
+            <li>
+              <a href="#"> <img alt="medium" src="./images/medium.png" /></a>
+            </li>
+            <li>
+              <a href="#"> <img alt="github" src="./images/github.png" /></a>
+            </li>
+            <li>
+              <a href="#"> <img alt="vec" src="./images/vec.png" /></a>
+            </li>
+          </ul>
+        </div>
+      </section>
+      <section class="scroll">
+        <div class="box-contain">
+          <ul>
+            <li class="ptflio-item">
+              <img
+                class="ptflio-image"
+                src="./images/Snapshoot-Portfolio.png"
+                alt="Portfolio"
+              />
+              <section>
```

Comment:
[OPTIONAL] 

-  I think you should consider replacing this `section` tag with a `div` tag because each project forms part of the works section. This is necessary to make your code semantic. You can read more on the usage of section and div [here](https://thisthat.dev/div-vs-section/.)👍 ;